.github/workflows/release-sdk.yml

```diff
@@ -0,0 +1,45 @@
+# When a new release is created on GitHub, this workflow builds the SDK as an XCFramework,
+# generates the API documents, and pushes a CocoaPods update.
+name: Radar SDK iOS Release
+
+on:
+  # Triggered when a new release is tagged in GitHub.
+  release:
+    types: [ published ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+
+  build:
+    runs-on: macos-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Build, test, and analyze
+        run: xcodebuild clean build analyze test -workspace Example/Example.xcodeproj/project.xcworkspace -scheme RadarSDK -destination "platform=iOS Simulator,name=iPhone 11" | xcpretty
+      
+      - name: Build archive for iPhone simulator
+        run: xcodebuild archive -scheme RadarSDK -archivePath "RadarSDK-iphonesimulator.xcarchive" -sdk iphonesimulator SKIP_INSTALL=NO CODE_SIGN_IDENTITY= CODE_SIGNING_REQUIRED=NO
```

Comment:
is CODE_SIGN_IDENTITY intentionally left blank for these?

Reply:
I copied this more or less as-is from the `Makefile` that Nick uses for manual builds.